src/sagas/index.js

```diff
@@ -88,8 +88,10 @@ const VM_FETCH_ADDITIONAL_SHALLOW = [
 export const EVERYONE_GROUP_ID = 'eee00000-0000-0000-0000-123456789eee'
 
 export function* transformAndPermitVm (vm) {
+  const entityPermissionsToUserPermits = yield curryEntityPermissionsToUserPermits()
+
   const internalVm = Transforms.VM.toInternal({ vm })
-  internalVm.userPermits = yield entityPermissionsToUserPermits(internalVm)
+  internalVm.userPermits = entityPermissionsToUserPermits(internalVm)
```

Comment:
Based on previous suggestion:

```suggestion
  internalVm.userPermits = _entityPermissionsToUserPermits(internalVm)

```